Assets/Scripts/Models/World.cs

```diff
@@ -1063,24 +1068,42 @@ void ReadXml_Characters(XmlReader reader)
             {
                 int x = int.Parse(reader.GetAttribute("X"));
                 int y = int.Parse(reader.GetAttribute("Y"));
+                Character c;
```

Comment:
Please name this variable as character. At the end of method you already trying to remember what is "c"